pallets/vector/src/lib.rs

```diff
@@ -368,7 +383,7 @@ pub mod pallet {
 					.expect("Rotate verification key should be valid at genesis.");
 			RotateVerificationKey::<T>::set(Some(rotate_verification_key));
 
-			SyncCommitteePoseidons::<T>::insert(self.period, self.sync_committee_poseidon);
+			SyncCommitteeHashes::<T>::insert(self.period, self.sync_committee_hash);
```

Comment:
The mainnet genesis will be different using this new binary even if we use the same chain spec. 
I am not sure about side-effect of that.